test/SourceKit/CursorInfo/cursor_doc_comment_from_clang_safe_wrapper.swift

```diff
@@ -0,0 +1,32 @@
+// REQUIRES: swift_feature_SafeInteropWrappers
+
+// RUN: %empty-directory(%t)
+// RUN: split-file --leading-lines %s %t
+
+//--- header.h
+
+/** This comment contains `markup`.
+ *
+ * - And a list
+ */
+void testCDecl(const int * __attribute__((__counted_by__(len))) __attribute__((noescape)) p, int len);
+
+//--- module.modulemap
+
+module MyClangModule { header "header.h" }
+
+//--- test.swift
+
+import MyClangModule
+
+func test(_ s: Span<CInt>) {
+  // RUN: %sourcekitd-test -req=cursor -pos=%(line + 1):4 %s -- %s -I %t -enable-experimental-feature SafeInteropWrappers | %FileCheck %s
+  testCDecl(s)
+}
+
+// CHECK-LABEL: DOC COMMENT
```

Comment:
You probably want to check part of the symbol graph output here as well, since that's actually what ends up being used by DocC.